includes/payment-methods/class-wc-stripe-upe-payment-method-amazon-pay.php

```diff
@@ -42,7 +42,8 @@ public static function is_amazon_pay_enabled() {
 			return false;
 		}
 
-		$upe_enabled_method_ids = WC_Stripe_Helper::get_settings( null, 'upe_checkout_experience_accepted_payments' );
+		$wc_stripe_upe_payment_method_amazon_pay = new self();
+		$upe_enabled_method_ids = $wc_stripe_upe_payment_method_amazon_pay->get_upe_enabled_payment_method_ids();
```

Comment:
How about calling the parent method `parent::update_payment_method_configurations()`?

Reply:
Since `get_upe_enabled_payment_method_ids` is not a static function, we need to call from an instance and the parent is abstract.  Do you have something else in mind?